### ISSUE TITLE
Failed test 'error msg ok'

### ISSUE THREAD

**Issue by eserte** [typester/Test-RedisServer#12]
On my freebsd 13 smokers the test suite failed --- probably because the error message format changed in newer redis versions:
```
#   Failed test 'error msg ok'
#   at t/unknown_conf.t line 17.
#                   '*** failed to launch redis-server ***
# 
# *** FATAL CONFIG FILE ERROR (Redis 6.0.10) ***
# Reading the configuration file, at line 3
# >>> 'unknown_key unknown_val'
# Bad directive or wrong number of arguments
#  at t/unknown_conf.t line 11.
# '
#     doesn't match '(?^:\*\*\* FATAL CONFIG FILE ERROR \*\*\*)'
# Looks like you failed 1 test of 2.
t/unknown_conf.t .. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/2 subtests 
```

**Comment by eserte** [comment]
This also happens on debian/bullseye (easier to reproduce, as a docker image could be used here).

**Comment by duffee** [comment]
There's a change in the way the error message is reported
`*** FATAL CONFIG FILE ERROR (Redis 6.0.11) ***`
fails the regex test because of the extra server name and version number included.

I'll work up a PR for it tomorrow

**Comment by Songmu** [comment]
released as 0.22. Thank you for reporting.
